pyomo/contrib/alternative_solutions/tests/test_aos_utils.py

```diff
@@ -0,0 +1,292 @@
+import pytest
+
```

Comment:
Please instead use:
```
from pyomo.common import unittest as unittest
```
instead of `pytest`. `pytest` is accessible via: `unittest.pytest`